src/components/LeafletMap.js

```diff
@@ -504,6 +509,24 @@ class LeafletMap extends Component {
     });
 
     this.map.addLayer(markersCluster);
+
+    if (markersData.length !== 0) {
```

Comment:
An explicit condition is redundant here. 

![image](https://user-images.githubusercontent.com/6225122/73576463-5d701680-442f-11ea-8c56-92c3bc12bec4.png)

Reply:
there is case when no markers or when sites data are not returned, so the map is set to default

Reply:
So I see that you're now using `if (markersData.length > 0) { ...` here in place. 

You don't really need that either. But I'm going to approve all the same.